pkgs/development/python-modules/pglast/default.nix

```diff
@@ -43,5 +43,6 @@ buildPythonPackage rec {
     changelog = "https://github.com/lelit/pglast/raw/v${version}/CHANGES.rst";
     license = licenses.gpl3Plus;
     maintainers = with maintainers; [ marsam ];
+    mainProgram = "pgpp";
```

Comment:
Would it make sense to have `pgpp` with `toPythonApplication` in `all-packages.nix`?

Reply:
I'm not sure. I use this package mostly as sql parser, and I sometimes use the cli to check whether sql/plpgsql file is valid. Also, I don't think the cli its main intent, because while it tries preserve comments, sometimes it's not accurate as libpg_query doesn't carry their exact location.
But I don't have a strong opinion, feel free to add it if you find it useful.